src/Components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom';
+
+const NavBar = () => (
+  <div className="navContainer">
+    <h2>Bookstore CMS</h2>
+    <ul className="nav">
+      <li>
+        <Link to="/" className="link">
+          BOOKS
+        </Link>
+      </li>
+      <li>
+        <Link to="/Categories" className="link">
+          CATEGORIES
+        </Link>
```

Comment:
- [ ] When I click the _**Categories**_ link, it shows the `Not Found` Page. This is because you have a typo here. You have _**/Categories**_ here but in your `App` component, you have `/catagories`. Kindly fix the typo. 👍